spec/requests/api/v1/events/events_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'faker'
+
 describe 'GET /v1/events/:id' do
```

Comment:
I would wrap all the `describe`s here in their own `describe` block. We've been using something like:
```ruby
describe 'Event requests' do
  ...
end
```

It will allow for some nice things later on that you haven't encountered yet, like test specific methods.